pkg/k8sclient/k8sclient.go

```diff
@@ -32,14 +55,22 @@ func GetNodeByLabel(label string, kubeClient kubernetes.Interface) (v1.Node, err
 	case 1:
 		return nodeList.Items[0], nil
 	default:
-		for _, node := range nodeList.Items {
-			nodeCapacity, ok := node.Status.Capacity[LvCapacity]
-			if !ok {
-				continue
+		if selectorMethod == RR {
+			if roundRobinIterator >= len(nodeList.Items){
+				roundRobinIterator = roundRobinIterator % len(nodeList.Items)
 			}
-			if (&nodeCapacity).CmpInt64(maxCapacity) == 1 {
-				maxCapacity = (&nodeCapacity).Value()
-				returnNode = node
+			returnNode = nodeList.Items[roundRobinIterator]
+			roundRobinIterator++
+		} else {
```

Comment:
let's make it future proof, and explicitly check for the currently supported other case as well

let's call that case "bin packing"